channels/serializers.py

```diff
@@ -229,6 +231,14 @@ def get_upvoted(self, instance):
         """Is a comment upvoted?"""
         return instance.likes is True
 
+    def get_profile_image(self, instance):
+        """Find the Profile for the comment author"""
+        profile = self.context['request'].user.profile
+        if (profile and profile.image_small):
+            return profile.image_small
+        else:
+            return "/static/images/avatar_default.png"
```

Comment:
We should get the static URL from Django so we can use cloudfront to serve this file. I found this code which is used by the Django static templatetag, it might work here:

    from django.contrib.staticfiles.storage import staticfiles_storage
    return staticfiles_storage.url(path)